frontend/src/employee-frontend/components/unit/TabApplicationProcess.tsx

```diff
@@ -0,0 +1,54 @@
+{
+  /*
+SPDX-FileCopyrightText: 2017-2020 City of Espoo
+
+SPDX-License-Identifier: LGPL-2.1-or-later
+*/
+}
+
+import React, { useContext } from 'react'
+import { ContentArea } from 'lib-components/layout/Container'
+import { UnitContext } from '../../state/unit'
+import { SpinnerSegment } from 'lib-components/atoms/state/Spinner'
+import ErrorSegment from 'lib-components/atoms/state/ErrorSegment'
+import Title from 'lib-components/atoms/Title'
+import { useTranslation } from '../../state/i18n'
+import _ from 'lodash'
+import TabPlacementProposals from './TabPlacementProposals'
+import TabWaitingConfirmation from './TabWaitingConfirmation'
+import TabApplications from './TabApplications'
+import { Gap } from 'lib-components/white-space'
+
+interface Props {
+  reloadUnitData: () => void
+}
+
+function TabApplicationProcess({ reloadUnitData }: Props) {
+  const { i18n } = useTranslation()
+
+  const { unitData } = useContext(UnitContext)
+
+  if (unitData.isLoading) {
+    return <SpinnerSegment />
+  }
+
+  if (unitData.isFailure || !unitData.value.placementPlans) {
+    return <ErrorSegment />
+  }
+
+  return (
```

Comment:
Use `renderResult` instead of iffitteling `isLoading/isFailure`

Reply:
Oops. Realized that after a previous refactor there is a bunch of unnecessary stuff here.